packages/fixie-web/src/voice.ts

```diff
@@ -212,6 +213,7 @@ export class VoiceSession {
     if (state != this._state) {
       console.log(`[voiceSession] ${this._state} -> ${state}`);
       this._state = state;
+      this.audioElement.muted = state != VoiceSessionState.SPEAKING;
```

Comment:
I think you're going to need some fudge factor here to prevent clipping the end of speech. Probably what this means is that we're going to need a different state to signal interruptions vs regular speaking turn change (which will need a more graceful transition).

Reply:
Long term you may be right, but this seems to do the trick for now.

Reply:
Well... if it's in the client it's harder to tweak later. Plus this changes it for everyone regardless of enable_interruptions. 

Maybe include a separate argument regarding whether to stop play out that we can tweak independently from the server side?

Reply:
Then we can only set the stop-playout flag on an interruption.

Reply:
From the comments in the other issue, it sounds like clipping will be a corner case, so I'm OK to revisit this later/if-needed.